telegram/ext/updater.py

```diff
@@ -74,7 +79,13 @@ def __init__(self, token=None, base_url=None, workers=4, bot=None):
         if bot is not None:
             self.bot = bot
         else:
-            self.bot = Bot(token, base_url)
+            # we need a connection pool the size of:
+            # * for each of the workers
+            # * 1 for Dispatcher
+            # * 1 for polling Updater (even if webhook is used, we can spare a connection)
+            # * 1 for JobQueue
+            self._request = Request(con_pool_size=workers + 3)
```

Comment:
@jh0ker 
I don't think we need it... but it probably doesn't hurt to add another connection.